src/coredump/coredumpctl.c

```diff
@@ -386,9 +386,6 @@ static int parse_argv(int argc, char *argv[]) {
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "--since= must be before --until=.");
 
-        if ((!!arg_directory + !!arg_image + !!arg_root) > 1)
-                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Please specify either --root=, --image= or -D/--directory=, the combination of these options is not supported.");
-
```

Comment:
This also drops the check if --root and --image are specified together, which should still be disallowed